lib/bundler/cli/add.rb

```diff
@@ -2,34 +2,45 @@
 
 module Bundler
   class CLI::Add
+    attr_reader :gems, :options, :version
+
     def initialize(options, gems)
       @gems = gems
       @options = options
-      @options[:group] = @options[:group].split(",").map(&:strip) if !@options[:group].nil? && !@options[:group].empty?
+      @options[:group] = options[:group].split(",").map(&:strip) if !options[:group].nil? && !options[:group].empty?
```

Comment:
I guess since we are at it we can parse this just like we parse `@version` in the following line.

```suggestion
      @options[:group] = options[:group].split(",").map(&:strip) unless options[:group].nil?
```

Reply:
Agree, also I noticed some params are parsing in some commands at the initializer (duplicated code), I think it's a good idea to move to precommand action.